src/redux/statsReducer.ts

```diff
@@ -32,63 +32,17 @@ const initialState: StatsState = {
 };
 
 export const statsReducer = createReducer(initialState, (builder) => {
-    builder.addCase(
-        StatsAction.HydrateMatchHistoryComplete,
-        (state, action) => {
-            const matchesCollection = action.payload;
-            state.matches = matchesCollection;
-            state.commanders = matchesToCommanders(matchesCollection);
-            state.players = matchesToPlayers(matchesCollection);
-        },
-    );
+    builder.addCase(StatsAction.HydrateMatchHistoryComplete, (state, action) => {
+        const matchesCollection = action.payload;
+        state.matches = matchesCollection;
+        state.commanders = matchesToCommanders(matchesCollection);
+        state.players = matchesToPlayers(matchesCollection);
+    });
 });
 
 // given a collection of matches, return all of the commanders in those matches

```

Comment:
you are correct-- these functions actually don't really do us any good anymore-- we can call the dictionaryHelpers directly in the statsReducer. Can you update that?

Reply:
I'm not sure I understand this request. The dictionaryHelpers are called within statsReducer currently.
Do you mean something like: replace all calls to these functions with calls directly to their helpers then remove these functions?

Reply:
Yes, exactly that